api/v1beta2/bgppeer_types.go

```diff
@@ -55,6 +59,11 @@ type BGPPeerSpec struct {
 	// +optional
 	KeepaliveTime metav1.Duration `json:"keepaliveTime,omitempty"`
 
+	// Requested BGP connect time, controls how long BGP waits between connection attempts to a neighbor.
+	// +kubebuilder:default:="120s"
```

Comment:
what i don`t like about this file that i didn't find a way to that the default value will appear only once in this file instead of twice.